packages/usehooks-ts/src/useClickAnyWhere/useClickAnyWhere.ts

```diff
@@ -0,0 +1,15 @@
+import { useEffect } from 'react'
+
+type AnyEvent = MouseEvent | TouchEvent
+type Handler = (event: AnyEvent) => void
+
+function useClickAnyWhere(handler: Handler) {
+  useEffect(() => {
+    const callback = (event: AnyEvent) => handler(event)
+
+    document.addEventListener('click', callback)
+    return () => document.removeEventListener('click', callback)
```

Comment:
Here, two things:
- We can simplify the hook using the `useEventListener`.
- The Typescript types check doesn't pass, we have to type it manually. 

```suggestion
    useEventListener('click', event => {
      handler(event as unknown as MouseEvent)
    })
```

Reply:
Good idea!